Mathlib/Geometry/Convex/Cone/Pointed.lean

```diff
@@ -221,4 +222,22 @@ lemma to_isOrderedModule (C : PointedCone R E) (h : ∀ x y : E, x ≤ y ↔ y -
     IsOrderedModule R E := .of_smul_nonneg <| by simp +contextual [h, C.smul_mem]
 
 end OrderedAddCommGroup
+
+section Salient
+variable [Semiring R] [PartialOrder R] [IsOrderedRing R] [AddCommGroup E] [Module R E]
+open Pointwise
+
+/-- A pointed cone is salient iff an element `x` being a member of both the cone and its negative
+is equivalent to `x = 0`. -/
+lemma salient_iff_mem_neg_eq_zero (C : PointedCone R E) :
+    (C : ConvexCone R E).Salient ↔ ∀ x, (x ∈ C ∧ x ∈ -C) ↔ x = 0 := by
```

Comment:
Probably this is more useful in practice, no? As otherwise you end up constantly needing to prove the trivial fact that `0 ∈ C ∧ 0 ∈ -C`.
```suggestion
    (C : ConvexCone R E).Salient ↔ ∀ x, (x ∈ C ∧ x ∈ -C) → x = 0 := by
```
Do you also want to have an `alias` for the reverse direction?

Reply:
Okay, sure, remove the version I suggested.